### DISCUSSION TITLE
Documentation - Add typescript examples

### DISCUSSION THREAD

**Discussion by ValterSantosMatos** [vercel/next.js#19809] [Ideas]
# Documentation - Add typescript examples

Love the NextJs docs but its always a nightmare to find the correct types to use with any of the examples, and end up spending more time searching for types than trying out a new feature. Not sure if its an isolated case but would be extremely helpful to have a version of the docs written in typescript.

Ty

**Comment by denistsoi** [comment]
- https://nextjs.org/docs/basic-features/typescript
Types.
- https://github.com/vercel/next.js/tree/canary/packages/next/types
